tracer/src/Datadog.Tracer.Native/debugger_method_rewriter.cpp

```diff
@@ -211,6 +212,7 @@ HRESULT DebuggerMethodRewriter::LoadInstanceIntoStack(FunctionInfo* caller, bool
 
 HRESULT DebuggerMethodRewriter::Rewrite(RejitHandlerModule* moduleHandler, RejitHandlerModuleMethod* methodHandler)
 {
+    // std::this_thread::sleep_for(std::chrono::milliseconds(4600));
```

Comment:
```suggestion
```

Reply:
Fixed in 5a103fbe57cedbc9e13568d4372c82a921c46e26